app/src/main/java/com/github/wnder/TemporaryActivity.java

```diff
@@ -0,0 +1,92 @@
+package com.github.wnder;
+
+import android.content.Context;
+import android.content.Intent;
+import android.location.LocationManager;
+import android.os.Bundle;
+
+import androidx.appcompat.app.AlertDialog;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.github.wnder.guessLocation.GuessLocationActivity;
+import com.github.wnder.networkService.NetworkService;
+import com.github.wnder.picture.PicturesDatabase;
+import com.github.wnder.tour.FirebaseTourDatabase;
+import com.github.wnder.tour.TourDatabase;
+import com.github.wnder.user.GlobalUser;
+import com.github.wnder.user.User;
+
+import javax.inject.Inject;
+
+import dagger.hilt.android.AndroidEntryPoint;
+
+@AndroidEntryPoint
+public class TemporaryActivity extends AppCompatActivity {
+
+    @Inject
+    public NetworkService networkInfo;
+
+    public TourDatabase tourDb;
+
+    @Inject
+    public PicturesDatabase picturesDb;
+
+    public static final double DEFAULT_LAT = 46.5197;
+    public static final double DEFAULT_LNG = 6.5657;
```

Comment:
This two constants are already declared in GuessPreviewActivity. You can change to keep them only in one class if it makes sense.